.env.example

```diff
@@ -36,8 +36,10 @@ CRYPTOCOMPARE_EXCEPTION_FREQUENCY=60
 
 SENTRY_LARAVEL_DSN=
 
-ARKSCAN_SUPPORT_ENABLED=false
 ARKSCAN_MIGRATION_ADDRESS=
+ARKSCAN_MIGRATION_WALLET_NAME=
```

Comment:
we have a `known_wallets` file for names (see [repo](https://github.com/ArkEcosystem/common)), that should allow it to show up with a custom name without needing to mimic delegate attributes like the PR currently does

Reply:
is also a separate task to look at that part down the line when more of the UI is in place